### ISSUE TITLE
Dataset Division

### ISSUE THREAD

**Issue by Huster-Hq** [golnarkmahani/Weakly-Supervised-Segmentation#2]
Could you please let me know the specific division of training set, verification set and test set？
